lib/lighthouse/letters_generator/service.rb

```diff
@@ -85,7 +85,7 @@ def download_letter(icn, letter_type, options = {})
         end
 
         endpoint = "letters/#{letter_type}/letter"
-        letter_options = options.select { |_, v| v == true }
+        letter_options = options.select { |_, v| v == 'true' }
```

Comment:
Why make this a string?

Reply:
Yeah, they come in as strings from the url. So it would have to be cast into a boolean.